js/okex.js

```diff
@@ -74,23 +76,68 @@ module.exports = class okex extends okcoinusd {
 
     async fetchTickers (symbols = undefined, params = {}) {
         await this.loadMarkets ();
+        let market = undefined;
         let request = {};
-        let response = await this.publicGetTickers (this.extend (request, params));
-        let tickers = response['tickers'];
-        let timestamp = parseInt (response['date']) * 1000;
+        if (undefined !== symbols)
+        {
```

Comment:
This bracket has to be on the same line with the `if` above, otherwise it will most likely break the transpiler...